resources/css/style.css

```diff
@@ -0,0 +1,117 @@
+* {
+    padding: 0;
+    margin: 0;
+    font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
+}
+.logo {
+    width: 50px;
+}
+
+body {
+background-image: url("/resources/images/20.jpg");
+background-size: cover;
+
+background-color: azure;
+}
+
+nav{
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    padding: 5px 20px;
+    height: 12vh;
+    background-color: bisque;
+    box-shadow: 0 0 10px black;
+
+}
+
+.heading{
+    display: flex;
+    align-items: center;
+    gap: 10px;
+}
+
+ul{
+        display: flex;
+        align-items: center;
+        gap: 35px
+        ;
+    }
+
+li{
+   list-style: none;
+ }
+
+a{
+    text-decoration: none;
+    font-weight: 800;
+}
+
+a:hover{
+    text-decoration: underline;
+    transition: 1s;
+}
+
+.image{
+    width: 600px;
+    box-shadow: black;
+    border-radius: 50px 0px 80px 0px;
+}
+
+main{
+    display: flex;
+    align-items: center;
+    justify-content: space-around;
+    height: 78vh;
+    padding: 20px;
+}
+.description{
+    max-width: 380px;
+    color: aliceblue;
+}
+h1{
+    font-size: 60px;
+    color: rgb(64, 41, 217);
+    text-shadow: 8px, 8px, 2px, 10px rgb(230, 230, 230);
+}
+
+.pri_btn{
+    padding: 15px 70px;
+    background-color: rgb(64, 41, 217);
+    border: none;
+    border-radius: 5px;
+    margin-top: 20px;
+    font-size: 20px;
+}
+
+footer{
+text-align: center;
+color: azure;
+background-color: rgb(64, 41, 217);
+font-size: 10px;
+height: 20px;
+padding-top: 10px;
+box-shadow: azure;
+}
+
+.btn{
+    background-color: rgb(64, 41, 217);
+    border: none;
+    border-radius: 8px;
+    color: azure;
+    padding: 5px 15px;
+    cursor: pointer;
+
+}
```

Comment:
Add an empty line after the closing curly brace

Reply:
alright sir